### PR TITLE
docs: add retry instructions to release process

### PR DESCRIPTION
**Pull request by hunterloftis**

### CODE REVIEW

File: RELEASE_PROCESS.md [hunk 1]
```diff
@@ -17,6 +17,12 @@ Users with permissions in the `@spectrum-web-components` organization on NPM can
 
 The docs site will publish automatically if the `#publish` string is included in the commit message and the check suite runs successfully.
 
+In the case of a publishing error, `yarn lerna-publish` will no longer work. Instead, you'll need to run:
```
Comment (Westbrook) [inline RELEASE_PROCESS.md:20]: If there is a failure _after_ tags have been published to GitHub?

Comment (hunterloftis) [reply RELEASE_PROCESS.md:20]: Possibly add a check like:

- visit https://github.com/adobe/spectrum-web-components
- see if "(your username) chore: release new versions" is the most-recent commit?

Comment (Westbrook) [reply RELEASE_PROCESS.md:20]: That or check: https://github.com/adobe/spectrum-web-components/tags for recent versions, but either work for me.

Comment (Westbrook) [reply RELEASE_PROCESS.md:20]: Ping for tomorrow. 🙏🏼

Comment (hunterloftis) [reply RELEASE_PROCESS.md]: @Westbrook how's that?

**Review by Westbrook — APPROVED** [review]
Ship it!

**Pull request merged**